### ISSUE TITLE
Particle effects don't work outside dev env

### ISSUE THREAD

**Issue by DrasticDemise** [baileyholl/Aura-Cascade#26]
HOW?

**Comment by DrasticDemise** [comment]
Currently attempted:
Updating forge to 1907 - Why would this ever work
Changing sideonly to isRemote - Nope.
Removing conditional particle bypassing - not that this would ever matter

So tilted

**Comment by williewillus** [comment]
Bad reflection (unupdated obfuscated names?)

**Comment by DrasticDemise** [comment]
Nothing about the particle or render code has been changed from 1.8.9 to the 1.9 build other than needing to transform access to the EntityFX class because some of the particle code was moved around, and I am not really seeing any type of reflection or otherwise questionable methods for particle rendering that would cause them to not appear in a regular client but work perfectly fine in the dev client (although it is extremely likely I could be glossing over it after looking at this for a few hours). I did tear apart and redo the gradle script, is it possible that when building gradle is referencing things it should not?

The packet system also seems to be working fine as the mirror still spawns particles that use the packet burst system.
